### PR TITLE
Updated the templates to handle FSx

### PR DESCRIPTION
**Pull request by spatel-gfb**
Updated the templates for handling SSD and HDD types of FSx for lustre file systems

Added FSx security group update to gwfcore root templete to include Batch Security Group.

Added Tag as Application : AWS-GENOMICS-WKF , for easy cost identification in billing

### CODE REVIEW

File: src/templates/gwfcore/gwfcore-code.template.yaml [hunk 1]
```diff
@@ -209,7 +209,9 @@ Resources:
       Tags:
         - Key: architecture
           Value: !FindInMap ["TagMap", "default", "architecture"]
-  
+        - Key: Application
+          Value: AWS-GENOMICS-WKF
```
Comment (henriqueribeiro) [inline src/templates/gwfcore/gwfcore-code.template.yaml]: what is this WFK? :)

File: src/templates/gwfcore/gwfcore-efs.template.yaml [hunk 2]
```diff
@@ -38,8 +38,10 @@ Resources:
       PerformanceMode: generalPurpose
       Encrypted: true
       FileSystemTags:
-      - Key: Name
-        Value: SharedDataGenomics
+        - Key: Name
+          Value: SharedDataGenomics
+        - Key: Application
+          Value: AWS-GENOMICS-WKF
```
Comment (henriqueribeiro) [inline src/templates/gwfcore/gwfcore-efs.template.yaml]: I don't think we should change any file we are not using, but it's just my opinion

File: src/templates/gwfcore/gwfcore-launch-template.template.yaml [hunk 3]
```diff
@@ -199,8 +202,18 @@ Resources:
...
               - fi
 
-              --==BOUNDARY==--
+              # optional FSx mount
+              - FSxId=`aws ssm get-parameter --name /gwfcore/${Namespace}/fsx-file-system --query 'Parameter.Value' --output text`
+              - if [[ $FSxId != "none" ]]
+              - then
+              - FSxMount=`aws ssm get-parameter --name /gwfcore/${Namespace}/fsx-mount-name --query 'Parameter.Value' --output text`
+              - mkdir -p /fsx && chmod 777 /fsx
+              - sudo amazon-linux-extras install -y lustre2.10
```
Comment (henriqueribeiro) [inline src/templates/gwfcore/gwfcore-launch-template.template.yaml:209]: is this not included on the lustre-client from above?

File: src/templates/gwfcore/gwfcore-code.template.yaml (hunk 1 above)
Comment (spatel-gfb) [reply src/templates/gwfcore/gwfcore-code.template.yaml]: it is short of workflow :)

**Review by henriqueribeiro — APPROVED** [review]

**Pull request merged**